integration/eth2network/config.yml

```diff
@@ -25,7 +25,7 @@ MAX_WITHDRAWALS_PER_PAYLOAD: 16
 DENEB_FORK_EPOCH: 0
 DENEB_FORK_VERSION: 0x20000093
 
-ELECTRA_FORK_VERSION: 0x20000094
+#ELECTRA_FORK_VERSION: 0x20000094
```

Comment:
was it important to comment this out for some reason? Maybe worth a comment

Reply:
I'll get rid of it. I noticed now that the beacon startup was complaining about it. Probably benign